src/utils/html.utils.ts

```diff
@@ -0,0 +1,36 @@
+export const setupContainer = (container: HTMLElement) => {
+  container.style.position = 'relative';
+  const style = getComputedStyle(container);
+  if (!style.display) {
+    container.style.display = 'block';
+    console.warn("[Orb] Graph container doesn't have defined 'display' property. Setting 'display' to 'block'...");
+  }
+  if (!style.width || style.width === '0px') {
```

Comment:
Is this covering cases like `'0rem'`, `'0'`, `'0 px'` if those are possible values for `.width` and `.height` (down below)?

Reply:
Good point. In firefox the `getComputedSize` returns a `0px` specifically. I can check if it `startsWith('0')` or something like that.

Another point is, what if the user programmatically is switching sizes, then we override that with this condition.

Reply:
Yep, that should do the trick. I just tried this:

```html
<div id="testing-height" style="height: 010px"></div>
```

```js
document.getElementById("testing-height").style.height // outputs '10px'
```

All in all, `010px` is a valid style, but luckily browser will take out the leading zeros.

Regarding user switching sizes, could we maybe then add settings param for it? Which is enabled by default. What do you think?

Reply:
@tonilastre That definitely makes sense! Justin Case - I added a regex to capture all "collapsed" style dimensions.

I've been meaning to talk to you about our settings. I'm not very happy with the current implementation. It's tricky to consume settings from a DX perspective. Plus, now they have duplicated fields. We could add a base interface for ViewSettings in the future.

Reply:
I agree with you regarding settings. I think we should do that along with the following fix: https://github.com/memgraph/orb/issues/16. With that in mind, we could create a baseline interface as well as `AbstractView`.